internal/cmd/run/cypress.go

```diff
@@ -136,7 +137,10 @@ func runCypressInDocker(p cypress.Project, testco testcomposer.Client, rs resto.
 	log.Info().Msg("Running Cypress in Docker")
 	printTestEnv("docker")
 
-	cd, err := docker.NewCypress(p, &testco, &testco, &rs, &rs, createReporters(p.Reporters))
+	reporters := createReporters(p.Reporters)
+	reporters = append(reporters, createSlackReporter(p.Notifications, p.Sauce.Metadata, &testco,
```

Comment:
Hm, spaghetti code 🍝 . Now we have two places where reporters are created.
Why not enhance `createReporters()` and simply pass in the required parameters. That'll keep everything in one place.

Reply:
done